packages/web-twig/src/Resources/components/Toast/stories/ToastDynamicToastQueue.twig

```diff
@@ -51,22 +97,19 @@
 </form>
 <!-- Toast and visual keyboard demo: end -->
 
-<!-- Toast show and hide demo: start -->
-<fieldset style="border: 0;">
-    <legend class="mb-500">Show the toast prepared in the DOM:</legend>
-    <Button
-        data-spirit-toggle="toast"
-        data-spirit-target="#my-hidden-dismissible-toast"
-        onclick="this.setAttribute('disabled', 'disabled')"
-        aria-expanded="false"
-    >
-        Show the hidden toast
-    </Button>
-</fieldset>
-<!-- Toast show and hide demo: end -->
+<Icon name="check-plain" isSymbol />
+<Icon name="danger" isSymbol />
+<Icon name="info" isSymbol />
+<Icon name="warning" isSymbol />
```

Comment:
Good job with these! 👏🏻 

Maybe I was wondering about moving the responsibility of hiding the SVG to their wrapper so it's immediately obvious these icons are not rendered to be visible:

```html
<div hidden>
  <!-- Now there can be `hidden` attribute instead of `style` – apart from `<svg>`, it's unlikely that `<div>`s will have `display` set somewhere in CSS. -->
  <Icon name="check-plain" isSymbol />
  <Icon name="danger" isSymbol />
  <Icon name="info" isSymbol />
  <Icon name="warning" isSymbol />
</div>
```

Then the inline `style` could be removed from the `inlineSvg()` function output.

What do you think?

Reply:
ok, why not :)